packages/dd-trace/test/appsec/reporter.spec.js

```diff
@@ -350,6 +355,43 @@ describe('reporter', () => {
     })
   })
 
+  describe('reportFingerprints', () => {
+    it('should not call addTags if parameter is undefined', () => {
+      Reporter.reportDerivatives(undefined)
+      sinon.assert.notCalled(span.addTags)
+    })
+
+    it('should call addTags with an empty array', () => {
+      Reporter.reportDerivatives([])
+      sinon.assert.calledOnceWithExactly(span.addTags, {})
+    })
```

Comment:
duplicate tests with just above

Reply:
should we merge reportFingerprints and reportSchemas ? it's just one function now. Up to you